src/goats_cli/cli.py

```diff
@@ -15,58 +14,13 @@
 from goats_cli.exceptions import GOATSClickException
 from goats_cli.modify_settings import modify_settings
 from goats_cli.process_manager import ProcessManager
-
-
-def _run_migrations(manage_file: Path) -> None:
-    """
-    Run Django database migrations.
-
-    Executes ``manage.py migrate`` for the project.
-
-    Parameters
-    ----------
-    manage_file : pathlib.Path
-        Path to the project's ``manage.py``.
-
-    Raises
-    ------
-    subprocess.CalledProcessError
-        If the underlying command exits with a non-zero status.
-    """
-    utils.display_info("Applying database migrations... ")
-    subprocess.run(
-        [f"{manage_file}", "migrate"],
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        check=True,
-    )
-    utils.display_ok()
-
-
-def _verify_manage_py(manage_file: Path, project_name: str) -> None:
-    """
-    Verify that ``manage.py`` exists for the project.
-
-    Parameters
-    ----------
-    manage_file : pathlib.Path
-        Absolute path to the project's ``manage.py``.
-    project_name : str
-        Project name used to improve the error message.
-
-    Raises
-    ------
-    GOATSClickException
-        If ``manage.py`` is not present at the expected location.
-    """
-    utils.display_info("Verifying 'manage.py' exists... ")
-    if not manage_file.is_file():
-        utils.display_failed()
-        raise GOATSClickException(
-            f"The 'manage.py' file for '{project_name}' does not exist at "
-            f"'{manage_file.absolute()}'."
-        )
-    utils.display_ok()
+from goats_cli.project_ops import run_migrations, verify_manage_py
+from goats_cli.servers import (
+    start_background_workers,
+    start_django_server,
+    start_redis_server,
+)
```

Comment:
Let's leave the refactoring for another ticket and keep the scope to just versioning.